### PR TITLE
[full-ci] [tests-only] Added test for creating group with case sensitive

### PR DESCRIPTION
**Pull request by grgprarup**
## Description
Group names are case insensitive with oCIS, we cannot create groups with the same name but different lower and upper cases. So, tests to check group creation with different lower and upper cases are added as suggested in https://github.com/owncloud/ocis/issues/4291#issuecomment-1210956364

## Related Issue

- Part of https://github.com/owncloud/ocis/issues/4291
- Related to https://github.com/owncloud/ocis/issues/4291#issuecomment-1210956364

## How Has This Been Tested?

- Locally
- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/bootstrap/GraphContext.php [hunk 1]
```diff
@@ -428,6 +428,32 @@ public function adminHasAddedUserToGroupUsingTheGraphApi(
 		}
 	}
 
+	/**
+	 * @When /^the administrator creates a group "([^"]*)" using the GraphApi$/
```
Comment (kiranparajuli589) [inline tests/acceptance/features/bootstrap/GraphContext.php:432]:
```suggestion
	 * @When /^the administrator creates a group "([^"]*)" using the GraphAPI$/
```
API: Application Programming Interface

File: tests/acceptance/features/apiGraph/createGroupCaseSensitive.feature [hunk 2]
```diff
@@ -0,0 +1,21 @@
+@api
+Feature: create groups, group names are case insensitive
+
+  Scenario Outline: group names are case insensitive, creating a groups with different upper and lower case names
+    Given using OCS API version "<ocs_api_version>"
+    And group "<group_id1>" has been created
+    When the administrator creates a group "<group_id2>" using the GraphApi
+    And the administrator creates a group "<group_id3>" using the GraphApi
+    Then the HTTP status code of responses on all endpoints should be "500"
```
Comment (individual-it) [inline tests/acceptance/features/apiGraph/createGroupCaseSensitive.feature:9]: is `500` really the reply we want?

Comment (grgprarup) [reply tests/acceptance/features/apiGraph/createGroupCaseSensitive.feature:9]: `500` is the actual current behavior of graph API for creating an existing user or group, as mentioned in comment https://github.com/owncloud/ocis/issues/3516#issuecomment-1098952180 the expected code must be `4XX`
We have an issue reported here https://github.com/owncloud/ocis/issues/3516 and mentioned the behavior in https://github.com/owncloud/ocis/issues/3247.

File: tests/acceptance/features/apiGraph/createGroupCaseSensitive.feature [hunk 3]
```diff
@@ -0,0 +1,21 @@
+@api
+Feature: create groups, group names are case insensitive
+
+  Scenario Outline: group names are case insensitive, creating a groups with different upper and lower case names
```
Comment (amrita-shrestha) [inline tests/acceptance/features/apiGraph/createGroupCaseSensitive.feature:4]:
```suggestion
  Scenario Outline: group names are case insensitive, creating groups with different upper and lower case names
```

**Review by individual-it — APPROVED** [review]

File: tests/acceptance/expected-failures-localAPI-on-OCIS-storage.md [hunk 4]
```diff
@@ -25,3 +25,5 @@ The expected failures in this file are from features in the owncloud/ocis repo.
 ### [Changing personal drive quota on another user as admin is not possible](https://github.com/owncloud/ocis/issues/4325)
 - [apiSpaces/changeSpaces.feature:221](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/changeSpaces.feature#L221)
 
+### [create request for already existing user exits with status code 500 ](https://github.com/owncloud/ocis/issues/3516)
+- [apiGraph/createGroupCaseSensitive.feature:4](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiGraph/createGroupCaseSensitive.feature#L4)
```
Comment (individual-it) [inline tests/acceptance/expected-failures-localAPI-on-OCIS-storage.md:29]: you have to mention every line of the examples table, not the line of the outline

**Review by kiranparajuli589 — APPROVED** [review]
lgtm :+1:

**Review by individual-it — APPROVED** [review]

File: .drone.env [hunk 5]
```diff
@@ -1,5 +1,5 @@
 # The test runner source for API tests
-CORE_COMMITID=f79fbff0a0d129df7b6ec20bc202665c4fda5fad
+CORE_COMMITID=73b5fce6b0181cd3989ca4af90cbcf0164c9c78b
```
Comment (amrita-shrestha) [inline .drone.env:2]: just to remember while merging change to latest

File: tests/acceptance/expected-failures-API-on-OCIS-storage.md [hunk 6]
```diff
@@ -742,17 +742,6 @@ Scenario Outline: Moving a file into a shared folder as the sharee and as the sh
...
-- [apiShareCreateSpecialToShares2/createShareGroupCaseSensitive.feature:86](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiShareCreateSpecialToShares2/createShareGroupCaseSensitive.feature#L86)
-- [apiShareCreateSpecialToShares2/createShareGroupCaseSensitive.feature:87](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiShareCreateSpecialToShares2/createShareGroupCaseSensitive.feature#L87)
-- [apiShareCreateSpecialToShares2/createShareGroupCaseSensitive.feature:88](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiShareCreateSpecialToShares2/createShareGroupCaseSensitive.feature#L88)
-- [apiShareCreateSpecialToShares2/createShareGroupCaseSensitive.feature:89](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiShareCreateSpecialToShares2/createShareGroupCaseSensitive.feature#L89)
-- [apiShareCreateSpecialToShares2/createShareGroupCaseSensitive.feature:90](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiShareCreateSpecialToShares2/createShareGroupCaseSensitive.feature#L90)
-
```
Comment (amrita-shrestha) [inline tests/acceptance/expected-failures-API-on-OCIS-storage.md:755]: done in BUMP PR

**Comment by grgprarup** [comment]
Need to rebase and push after bump PR https://github.com/owncloud/ocis/pull/4434 is merged.

**Review by phil-davis — APPROVED** [review]

**Pull request merged**